tests/unit_tests/strictload_enum_test.py

```diff
@@ -45,20 +46,20 @@ def setUpClass(cls):
         cls.checkpoint_diff_keys_name = 'strict_load_test_diff_keys.pth'
         cls.checkpoint_diff_keys_path = cls.temp_working_file_dir + '/' + cls.checkpoint_diff_keys_name
 
-        # Setup the model
+        # Setup the trainer
         cls.original_torch_net = Net()
 
-        # Save the model's state_dict checkpoint with different keys
+        # Save the trainer's state_dict checkpoint with different keys
         torch.save(cls.change_state_dict_keys(cls.original_torch_net.state_dict()), cls.checkpoint_diff_keys_path)
 
-        # Save the model's state_dict checkpoint in Trainer format
-        cls.trainer = Trainer("load_checkpoint_test", model_checkpoints_location='local')  # Saves in /checkpoints
-        cls.trainer.build_model(cls.original_torch_net, arch_params={'num_classes': 10})
+        # Save the trainer's state_dict checkpoint in Trainer format
+        cls.sg_trainer = Trainer("load_checkpoint_test", model_checkpoints_location='local')  # Saves in /checkpoints
```

Comment:
why did you change trainer to sg_trainer?

Reply:
renamed back.